### PR TITLE
Hotfix/upgrade es

### PR DESCRIPTION
**Pull request by rossmiller**
Upgrade packages to support Elasticsearch 7.9.

Update readme and add makefile with commonly used commands.

Remove search heisen tests.

### CODE REVIEW

**Review by dvadgama — APPROVED** [review]

File: Gemfile [hunk 1]
```diff
@@ -13,9 +13,9 @@ gem 'railties', RAILS_VERSION
 gem 'ancestry'
 gem 'carrierwave'
 gem 'country_select'
-gem 'elasticsearch', '~> 6.1' # TODO: Pinned because we're using ES 6.x, not 7.x
-gem 'elasticsearch-model', '~> 6.0.0' # TODO: Pinned due to search result order issue in >= 6.1
-gem 'elasticsearch-rails', '~> 6.1'
+gem 'elasticsearch', '~> 7.9.0'
```
Comment (davecharles) [inline Gemfile:16]: Is the pinning still necessary?

**Review by davecharles — COMMENTED** [review]
One comment, but lgtm

File: Gemfile (hunk 1 above)
Comment (rossmiller) [reply Gemfile:16]: It looks like it from the docs:

_The client's API is compatible with Elasticsearch's API versions from 0.90 till current, just use a release matching major version of Elasticsearch._

https://github.com/elastic/elasticsearch-ruby/tree/master/elasticsearch
https://github.com/elastic/elasticsearch-rails/tree/master/elasticsearch-model

7.9 is the version of ES which will be used in PAAS.

Comment (davecharles) [reply Gemfile:16]: 👍

**Review by davecharles — APPROVED** [review]
lgtm

**Pull request merged**